src/stream.jl

```diff
@@ -13,7 +13,8 @@ end
 function Stream()
     p = Ref{Ptr{Void}}()
     rt.cudaStreamCreate(p)
-    hnd = CuStream(p[])
+    ctx = CuCurrentContext()
```

Comment:
The contexts should be in https://github.com/JuliaGPU/CUDArt.jl/blob/8dd00425fc6de4850a42e85b2eff259f858bbcac/src/device.jl#L16 for anything that comes from `CUDArt` and goes to `CUDAdev` so that the memory management of `CUDAdev` does the right thing.